app/views/forms/show.html.erb

```diff
@@ -7,8 +7,11 @@
     <h1 class="govuk-heading-l">
       <span class="govuk-caption-l"><%= @form.name %></span><span class="govuk-visually-hidden"> - </span>
       <%= @form.live? ? t("forms.form_overview.live_title") : t("forms.form_overview.title") %>
+      <span class="govuk-!-display-block"><%= render FormStatusTagComponent::View.new(status: @form.status) %></span>
```

Comment:
You may have already tried this and found an issue, but instead of styling a `span` into a block element we could use a `div`here

Reply:
@DavidBiddle would a div inside a `h1` be valid markup?  https://caninclude.glitch.me/caninclude?child=div&parent=h1

Reply:
This is moot now anyway 😄